ipatests/test_integration/test_epn.py

```diff
@@ -724,6 +729,7 @@ def test_EPN_starttls(self, cleanupmail):
             validate_mail(self.master, i,
                           "Hi test user,\nYour login entry user%d is going" % i)
 
+    @skip_if_fips()
```

Comment:
I don't think that this skip is needed, the run from Nightly-STIG-Tier2/RHEL9.3/2023-09-15_00-42 shows a successful test.